### PR TITLE
Vagrantfile url

### PR DESCRIPTION
**Pull request by lutherling12**
- The corrected link downloads Qt5.5.1 tarball without 403 error.
- It seems the original intent was to save the tarball to the project root directory mounted at /vagrant, and the changed Vagrantfile reflects that.
- Echo statement assures user that provisioning script isn't hanging because of quiet wget.

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
@peterbarker Didn't you put up the original? If so can you look a this?

File: Vagrantfile [hunk 1]
```diff
@@ -24,14 +24,15 @@ Vagrant.configure(2) do |config|
      sudo apt-get dist-upgrade -y
      sudo apt-get install -y git build-essential
      sudo apt-get install -y espeak libespeak-dev libudev-dev libsdl1.2-dev
-     sudo apt-get install -y doxygen 
+     sudo apt-get install -y doxygen
      sudo apt-get install -y gstreamer1.0* libgstreamer1.0*
 
      # taken from travis.yml
-     su - vagrant -c 'wget --continue -q %{deps_url}/%{qt_deps_tarball}'
+     echo 'Saving Qt5.5.1-linux-tar.bz2 from %{deps_url} to %{project_root_dir}'
```
Comment (peterbarker) [inline Vagrantfile]: Should use  %{qt_deps_tarball} in place of Qt5.5.1-linux-tar.bz2

File: .vagrantconfig.yml [hunk 2]
```diff
@@ -18,4 +20,4 @@ configs:
         'spec': 'linux-g++-64'
         'shadow_build_dir': '/vagrant/shadow-build'
         'pro': '/vagrant/qgroundcontrol.pro'
-        'deps_url': 'https://s3-us-west-2.amazonaws.com/qgroundcontrol/dependencies'
+        'deps_url': 'https://s3-us-west-2.amazonaws.com/qgroundcontrol/dependencies/Qt5.5.1-linux.tar.bz2'
```
Comment (peterbarker) [inline .vagrantconfig.yml]: Should use %{qt_deps_tarball} in place of Qt5.5.1-linux-tar.bz2

**Comment by peterbarker** [comment]
On Fri, 17 Jun 2016, Don Gagne wrote:

> @peterbarker Didn't you put up the original? If so can you look a this?

Looks good apart from the line comments I've added.  Can be merged as-is.

Of course, the build *does* fail, but I believe for unrelated reasons :-)

**Comment by dagar** [comment]
Why did your build fail? Our other builds haven't failed recently.

https://travis-ci.org/mavlink/qgroundcontrol/branches

**Comment by peterbarker** [comment]
On Sat, 18 Jun 2016, Daniel Agar wrote:

> Why did your build fail? Our other builds haven't failed recently.

Looks like submodules got added to the build recently.  I assume 
.travis.yml was updated but Vagrantfile was not.

https://github.com/mavlink/qgroundcontrol/pull/3598

That PR incorporates lutherling12's fixes and adds a submodule init and 
update into the Vagrantfile.

**Comment by dagar** [comment]
Thanks. These commits were merged as part of #3598.

**Pull request closed without merging**